testCases/cypress/helpers/institute-set-up-utils.js

```diff
@@ -0,0 +1,25 @@
+//const {deleteInstituteSetUp,createDistrict,createSchool,createAuthorityWithContactToTest,createDistrictWithContactToTest,createSchoolWithContactToTest, clearSchoolContacts, setupSchoolContact} = require('../services/institute-api-service');
+import {OAuthUtil} from '../services/institute-api-service.cy';
+const instituteSetupUtils = {
+
+    async setUpDistrictAndSchool(){
+        debugger
+        await instituteSetupUtils.deleteInstituteSetUp();
+        //const districtID = await createDistrict();
+        console.log('AT district created');
+        // const school = await createSchool(districtID);
+        console.log('AT school created');
```

Comment:
we can use log.info here.

Reply:
Seeing a few flavours of logging - we should discuss.